### PR TITLE
ONNX dockerfile (build from source, for development)

### PR DESCRIPTION
**Pull request by vinitra**
Creating a lightweight ONNX dockerfile for developers to setup, develop and build ONNX from source. This is a way to improve the build/development experience of ONNX.

### CODE REVIEW

File: onnx-devel/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+FROM ubuntu:18.04
```
Comment (vinitra) [inline onnx-devel/Dockerfile]: Should we use an even more lightweight image?

Comment (vinitra) [reply onnx-devel/Dockerfile]: -> updated to python:3.7-slim

File: onnx-devel/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM python:3.7-slim
+
```
Comment (linkerzhang) [inline onnx-devel/Dockerfile]: the folder name may be either "onnx-dev" or "onnx-develop", I think.

Comment (vinitra) [reply onnx-devel/Dockerfile]: Renamed. Thanks!

**Pull request merged**